site/components/pages/Pledge/PledgeDashboard/Cards/FootprintCard/index.tsx

```diff
@@ -22,18 +22,8 @@ const PlaceholderFootprintChart: React.ComponentType = dynamic(
   { ssr: false }
 );
 
-const COLOR_PALETTE = [
-  "#147b11",
-  "#2c8e18",
-  "#4aa11f",
-  "#6db524",
-  "#96cb27",
-  "#c0dc30",
-  "#e3e541",
-  "#eedc51",
-  "#f6d562",
-  "#fdd175",
-];
+/** This palette is defined at https://www.figma.com/file/moDUIfGjQaISUGDjLxyko4/Multi-Wallet-Pledge?node-id=7%3A161 */
+const COLOR_PALETTE = ["#0BA1FF", "#E081EB", "#FF6F88", "#FFA600"].reverse();
```

Comment:
@theprofessora - we currently support a maximum of 10 different footprint categories on pledges.
Are you happy to reuse the colours here or would you extend the palette to include 10 unique colours?

Reply:
@0xAeterno @Atmosfearful 

I created a 10-color palette for all the categories here: https://www.figma.com/file/moDUIfGjQaISUGDjLxyko4?node-id=1:3#286886177

Our previous color palette of varying shades of green was not WCAG accessible, and part of our goal going forward is to maximize accessibility across the product

Reply:
Have updated the palette with the new colors. It is also possible to generate a multi-color gradient and then select points within that if we don't want to be limited to 10 colors (though there could be contrast issues once we get to a large number of categories)

Reply:
Definitely a good idea to explore if we plan to extend pledge footprint categories past 10.
Might not be the right time with defined limits and accessibility requirements.